docs/tutorials/performance/load-test-grpc.md

```diff
@@ -2,16 +2,16 @@
 template: main.html
 ---
 
-# Load test gRPC endpoint
+# Load test gRPC method
 
 Load test a Kubernetes gRPC service and visualizing the performance metrics with an Iter8 Grafana dashboard.
 
-See [Load Test multiple gRPC methods](./load-test-http-multiple.md) to see a tutorial that describes how to load test multiple methods from an gRPC service.
+See [Multiple gRPC methods](./load-test-http-multiple.md) for a tutorial that describes how to load test multiple methods from an gRPC service.
```

Comment:
I know the original name is a bit lengthy but do you think that it's more descriptive and better aligns with the title of this tutorial?

Reply:
I took this out. It seems weird to have a reference so early to a more complicated use case.